tasks/GiveMeAHand.tex

```diff
@@ -27,38 +27,44 @@ \subsection*{Setup}
+	\item The robot approaches the delivery area where there is intended location ambiguity and requests clarification.
+	\item The robot waits for user confirmation before proceeding with object placement.
+	\item The robot delivers the item and returns to the initial operator's location for a new object.
 \end{enumerate}
 
-
 \subsection*{Additional rules and remarks}
 \begin{enumerate}[nosep]
-	\item \textbf{HRI:}
+	\item \textbf{Handover:}
+	\begin{itemize}[nosep]
+		\item To ensure user safety, the robot must position its manipulator within 10-20 cm of the human's hand, without making physical contact, and must clearly indicate its readiness to receive the object using an explicit signal (e.g., holding position, activating a light, saying it, or playing a sound).
+		\item The human must complete the handover by placing the object into the robot's manipulator; the robot must remain stationary and must not initiate physical contact during this phase.
+		\item Explicit operator commands during the object handover—such as \textit{“Please, place the item in my gripper and wait until I close it in three, two, one”}—will be considered \textit{Deus ex Machina}. However, the robot may notify the user before closing its gripper after detecting that the object is within its grasping range; for example, \textit{“I have detected that the object is in my hand. I will close my gripper in three seconds—please be careful.”}
+	\end{itemize}
+	\item \textbf{Delivery:}
 	\begin{itemize}[nosep]
-		\item The operator instructs the robot using commands and gestures (e.g. \textit{"Place this item there (pointing), next to the flower base"}).
-		\item At any moment, the robot may ask questions to confirm the information, request more details, or ask the operator to repeat the instructions.
-		\item In this task, natural human-robot interaction is the primary focus. Before engaging with a human, the robot must first capture their attention -- for example, through visual signals, gestures, or sound cues -- rather than relying on unattended interactions (e.g., speaking without addressing a specific person). This is especially important when the human and the robot are not in close proximity.
+		\item Delivery instructions may include phrases like: \textit{"Place this apple on that plate"} or \textit{"Put this can in the trash bin."} The operator gives instructions using both verbal commands and gestures (e.g., \textit{"Place this item there} \textit{(pointing), next to the flower vase"}).
+		\item At any moment, the robot may ask questions to confirm information, request clarification, or ask the operator to repeat the instruction.
+		\item Since the target locations are intentionally ambiguous, the robot must request clarification from the operator before proceeding. This must be done through natural interaction, such as verbal questions or multimodal cues (e.g., pointing, asking “Which one?”). The robot must not guess or place the object without confirmation.
```

Comment:
```suggestion
		\item Since the target locations are intentionally ambiguous, the robot can request clarification from the operator before proceeding. This should be done through natural interaction, such as verbal questions or multimodal cues (e.g., pointing, asking “Which one?”). 
```